search/query_params.cpp

```diff
@@ -26,6 +26,10 @@ map<string, vector<string>> const kSynonyms = {
     {"sw",   {"southwest"}},
     {"se",   {"southeast"}},
     {"st",   {"saint", "street"}},
```

Comment:
@vng in Germany and Switzerland "str" is used for "strasse" (street).

Reply:
Should these synonyms check the language of the map region and use only corresponding translation? @TODO it in the issue?

Reply:
strasse (straße) is a special case and it's processed separately.
See AddStrasseNames in search_index_builder.cpp

Reply:
@bnitkin it is worth being added as a comment here.

Reply:
@todo is a good idea. I was thinking about matching languages, but don't know how to get the map region language from `QueryParams::AddSynonyms()`.